src/Sfa.Tl.Matching.Web.UnitTests/Controllers/Opportunity/When_Placement_Information_Loaded_With_Placements_Unknown.cs

```diff
@@ -16,22 +16,22 @@ public class When_Placement_Information_Loaded_With_Placements_Unknown
     {
         private readonly IActionResult _result;
 
-        private readonly OpportunityDto _dto = new OpportunityDto();
+        private readonly PlacementInformationSaveDto _dto = new PlacementInformationSaveDto();
         private const bool PlacementsKnown = false;
         private const int Placements = 5;
         private const int OpportunityId = 12;
 
         public When_Placement_Information_Loaded_With_Placements_Unknown()
         {
-            _dto.Id = OpportunityId;
+            _dto.OpportunityId = OpportunityId;
             _dto.PlacementsKnown = PlacementsKnown;
             _dto.Placements = Placements;
 
-            var config = new MapperConfiguration(c => c.AddProfiles(typeof(EmployerMapper).Assembly));
+            var config = new MapperConfiguration(c => c.AddProfiles(typeof(PlacementInformationSaveDtoMapper).Assembly));
```

Comment:
c.AddProfiles(typeof(PlacementInformationSaveDtoMapper).Assembly is Identical to 
c.AddProfiles(typeof(EmployerMapper).Assembly

both will pick up all the profiles from same assembly